src/components/study-container.jsx

```diff
@@ -255,7 +255,7 @@ export function StudyContainer({ view, onChangeTab }) {
             const updateTypeHeader = eventData.headers[UPDATE_TYPE_HEADER];
             const errorMessage = eventData.headers[ERROR_HEADER];
             const userId = eventData.headers[USER_HEADER];
-            if (userId !== userName) {
+            if (userId != null && userId !== userName) {
```

Comment:
With your new condition, every user will have a message if the user is not set (like a build failure). I think this is not something we always want.

Reply:
At least for now, it is something we always want (always being this one specific case because no other exists yet) or am I missing something ?